Makefile

```diff
@@ -0,0 +1,10 @@
+
+.PHONY: test build
+
+build:
+	go build ./...
+
+test:
```

Comment:
Should we remove the `test.sh` file?

Reply:
Yes, we can do that since we have this covered in the makefile